light_curves/code_src/ztf_functions.py

```diff
@@ -179,19 +176,17 @@ def load_lightcurves(locations_df, nworkers=6):
         for location in tqdm(location_grps):
             lightcurves.append(load_lightcurves_one_file(location))
         return pd.concat(lightcurves, ignore_index=True)
-    
+
     # if we get here, multiprocessing has been requested
 
-    # number of files to be sent in as one "chunk" of work
-    chunksize = 100
     # make sure the chunksize isn't so big that some workers will sit idle
     if len(location_grps) < nworkers * chunksize:
         chunksize = len(location_grps) // nworkers + 1
 
     # "spawn" new processes because it uses less memory and is thread safe (req'd for pd.read_parquet)
     # https://stackoverflow.com/questions/64095876/multiprocessing-fork-vs-spawn
     mp.set_start_method("spawn", force=True)
-    
+
```

Comment:
I'm glad all this whitespace is getting cleaned up as it's a bit frustrating to work with. I've had to turn off the whitespace auto trim functionality of my IDE in order to work in this repo, but now I'm contributing to the problem. 

I wonder if we should do a separate PR that cleans up the formatting for the whole light_curves/ directory but doesn't make any substantive changes to the code? It would be easier to review that way. In addition to whitespace, the line lengths in some of these files is very long. Running autopep8 would be nice.

Reply:
I did use a max-line-length of 120, so flake8 is now clear on this PR.

(with one exception of `light_curves/code_src/ztf_functions.py:20:60: W605 invalid escape sequence '\s'`, but I didn't want to touch the ZTF path logic in this PR knowing my other one has some changes for that already)

Reply:
> (with one exception of light_curves/code_src/ztf_functions.py:20:60: W605 invalid escape sequence '\s', but I didn't want to touch the ZTF path logic in this PR knowing my other one has some changes for that already)

Oh, this is just the separator -- telling pandas that the file is space delimited. Maybe it needs to be a raw string: `sep=r"\s+"` instead of `sep="\s+"`.